assets/css/screen.css

```diff
@@ -1162,16 +1457,198 @@ Usage (In Ghost editor):
+    margin: 0 0 3px;
+    font-size: 1.3rem;
+    font-weight: 500;
+    line-height: 1.4em;
+}
+
+.post-full-byline-meta h4 a {
+    color: color(var(--darkgrey) l(+10%));
+}
+
+.post-full-byline-meta h4 a:hover {
+    color: var(--darkgrey);
+}
+
+.author-avatar {
+    display: block;
+    overflow: hidden;
+    margin: 0 -4px;
+    width: 40px;
+    height: 40px;
+    border: #fff 2px solid;
+    border-radius: 100%;
+    transition: all 0.5s cubic-bezier(0.4, 0.01, 0.165, 0.99) 700ms;
+}
+
+.author-list-item .author-card {
+    position: absolute;
+    bottom: 130%;
+    left: 50%;
+    z-index: 600;
+    display: flex;
+    justify-content: space-between;
+    margin-left: -200px;
+    width: 400px;
+    font-size: 1.4rem;
+    line-height: 1.5em;
+    background: white;
+    border-radius: 3px;
+    box-shadow: rgba(39,44,49,0.08) 0 12px 26px, rgba(39, 44, 49, 0.06) 1px 3px 8px;
+    opacity: 0;
+    transition: all 0.35s cubic-bezier(0.4, 0.01, 0.165, 0.99);
+    transform: scale(0.98) translateY(15px);
+    pointer-events: none;
+}
+
+.author-list-item .author-card:before {
+    content: "";
+    position: absolute;
+    top: 100%;
+    left: 50%;
+    display: block;
+    margin-left: -8px;
+    width: 0;
+    height: 0;
+    border-top: 8px solid #fff;
+    border-right: 8px solid transparent;
+    border-left: 8px solid transparent;
+}
+
+.author-list-item .author-card.hovered {
+    opacity: 1.0;
+    transform: scale(1) translateY(0px);
+    pointer-events: auto;
+}
+
+.author-card {
+    padding: 20px 20px 22px;
+}
+
+.author-card .author-info {
+    flex: 1 1 auto;
+    padding: 0 0 0 20px;
+}
+
+.author-card .author-info h2 {
+    margin: 8px 0 0;
+    font-size: 1.6rem;
+}
+
+.author-card .author-info p {
+    margin: 4px 0 0;
+    color: color(var(--midgrey) l(-10%));
+}
+
+.author-card .author-info .bio h2 {
+    margin-top: 0;
+}
+
+.author-card .author-info .bio p {
+    margin-top: 0.8em;
+}
+
+.author-card .author-profile-image {
+    flex: 0 0 60px;
+    margin: 0;
+    width: 60px;
+    height: 60px;
+    border: none;
+}
+
+.basic-info .avatar-wrapper {
+    position: relative;
+    margin: 0;
+    width: 60px;
+    height: 60px;
+    border: none;
+    background: rgba(229, 239, 245, 0.1);
+}
+
+.basic-info .avatar-wrapper svg {
+    margin: 0;
+    width: 60px;
+    height: 60px;
+    opacity: 0.15;
+}
+
+@media (max-width: 1170px) {
+    .author-list-item .author-card {
+        margin-left: -50px;
+        width: 430px;
+    }
+
+    .author-list-item .author-card:before {
+        left: 50px;
+    }
+}
+
+@media (max-width: 650px) {
+    .author-list-item .author-card {
+        display: none;
+    }   
+}
+
+@media (max-width: 500px) {
+    .author-avatar {
+        width: 32px;
+        height: 32px;
+    }
+
+    .post-full-byline-meta {
+        font-size: 1.2rem;
+    }
+
+    .post-full-byline-meta h4 {
+        font-size: 1.2rem;
+        margin-bottom: 2px;
+    }
+}
+
+
+/* 7.2. Subscribe Form
```

Comment:
to be checked: do we keep the subscribe form? we'll replace subscriptions with members in 3.0, though this theme can be still used with 2.x installs...

Reply:
TBC this week